### ISSUE TITLE
Provide UI hint to user of "skip till end of turn" functionality in game

### ISSUE THREAD

**Issue by melvinzhang** [magarena/magarena#263]
Currently it is only documented in the README.txt. Would be useful to have an in-game hint to make it more discoverable. 

Triggered by https://github.com/magarena/magarena/issues/238#issuecomment-102207596

**Comment by ShawnieBoy** [comment]
Maybe having a split-button or dropdown option for skip till end of turn?

**Comment by lodici** [comment]
I like the split button idea, assuming you mean a button with a drop down arrow that sets the default action, something like...

![capture-14](https://cloud.githubusercontent.com/assets/2443519/7668886/2ef549d6-fc4c-11e4-928a-c68e2319683d.gif)

**Comment by ShawnieBoy** [comment]
Kinda like that, but not setting a default - saves any mis-clicks. Default should always be one action/event

**Comment by lodici** [comment]
How about...
![screenshot00162](https://cloud.githubusercontent.com/assets/2443519/7843857/7f887f20-04a4-11e5-992a-48f260ef055d.png)
by simply updating ``MESSAGE`` in ``MagicPlayChoice``.

**Comment by lodici** [comment]
In the end, went with...
![screenshot00163](https://cloud.githubusercontent.com/assets/2443519/7860026/7b232914-053d-11e5-9a7d-98a6eace4820.png)

**Comment by melvinzhang** [comment]
It is a bit odd that only the click option is mentioned for passing
priority, but all three options are mentioned for skip till end of turn.

**Comment by lodici** [comment]
![screenshot00164](https://cloud.githubusercontent.com/assets/2443519/7860266/6b134aa2-053f-11e5-85bd-f2c150052100.png)
?

**Comment by melvinzhang** [comment]
The difference in the two sentence still throws me off, press vs click, spacebar vs space. Perhaps
```
Left click -> or Space to pass priorty.
[Right click -> or Shift+Space to skip till end of turn]
```

**Comment by lodici** [comment]
TBH, I'm not fussed either way when it comes to the exact wording, although I do think having one keyboard shortcut (shift+space) is clearer. Maybe its a legacy thing that there is shift+space and shift+right arrow.

**Comment by ShawnieBoy** [comment]
What about something like this? (Really quickly and poorly done)
![priority box](https://cloud.githubusercontent.com/assets/2363180/7864424/cd32b4ce-0559-11e5-95b2-9d38af6c2ca1.png)

With the additional text to say "Press >| to skip till end of turn" Can leave the various shortcuts out.

**Comment by lodici** [comment]
I think if you are going to do that it needs to fit the two buttons into the same space taken by the -> button. And I don't think there is room. Also you would still need to display the keyboard shortcuts. The fact is there is this (relatively) large area devoted to the prompt message so why not use it?

**Comment by ShawnieBoy** [comment]
I was thinking that skipping to the end of turn would occur less than just passing priority, so would have a smaller button.

I agree that the others are a simple solution :grinning: Just feels a bit wordy for something you'll be seeing a lot of.

**Comment by melvinzhang** [comment]
I do agree that the additional line makes it seem more wordy. About keyboard shortcuts, I think a common idiom nowadays is to show them when the user press "?" on the keyboard. Would something like that be useful in making them more discoverable? Then we can keep the wording on the UI shorter.
